src/types/UiaTextRangeBase.cpp

```diff
@@ -938,6 +938,10 @@ try
     const auto maxLengthOpt = (maxLength == -1) ?
                                   std::nullopt :
                                   std::optional<unsigned int>{ maxLength };
+    _pData->LockConsole();
+    auto Unlock = wil::scope_exit([&]() noexcept {

```

Comment:
Instead of `&` this should probably use `this` instead.
I'd also suggest releasing the Unlock variable after you're done with the call.